ui/dashboards/src/components/Dashboard/Dashboard.tsx

```diff
@@ -26,16 +27,20 @@ export type DashboardProps = BoxProps & {
   emptyDashboardProps?: EmptyDashboardProps;
   panelOptions?: PanelOptions;
 };
+const HEADER_HEIGHT = 165; // Approximate height of the header in dashboard view (including the navbar and variables toolbar)
```

Comment:
Is this working if user zoom in the browser ? :smile:

Reply:
Well it's an approximate height for desktop screen, this value is here only while the ref is not loaded. I think I could even put 0 🤔